pkgs/tools/misc/ectool/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "ectool";
+  version = "4.9";
+
+  src = fetchurl {
+    url = "https://coreboot.org/releases/coreboot-${version}.tar.xz";
```

Comment:
If in the future another package is added for `coreboot` utils, it would probably be best if there was a package that was just all of them.

Reply:
Very valid point, I will look into it in the future.